docs/src/components/landing-page/MainLayoutHeader.vue

```diff
@@ -116,13 +118,25 @@ export default defineComponent({
     modelValue: {
       type: Boolean,
       default: false
+    },
+    footerAtTop: {
+      type: Boolean,
+      default: false
+    },
+    primaryHeaderHeight: {
```

Comment:
You don't really need `primaryHeaderHeight` and `secondaryHeaderHeight`, because from outside you don't need to know the component is composed by multiple toolbars, you only need the whole header height, which you can easily get by adding a template `ref` on the `MainLayoutHeader` from outside as you should do with QFooter 

Revert the changes adding these 2 props